packages/forms/src/elements/common/Field.tsx

```diff
@@ -24,26 +24,36 @@ export interface IFieldProps extends HTMLAttributes<HTMLDivElement> {
 export const Field = React.forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
   (props, ref) => {
     const [hasHint, setHasHint] = useState(false);
+    const [hasMessage, setHasMessage] = useState(false);
     const [isLabelActive, setIsLabelActive] = useState(false);
     const [isLabelHovered, setIsLabelHovered] = useState(false);
     const multiThumbRangeRef = useRef<HTMLDivElement>(null);
-    const getMessageProps = (messageProps: any) => ({ role: 'alert', ...messageProps });
-    const { getInputProps, ...propGetters } = useField(props.id);
+    const { getInputProps, getMessageProps, ...propGetters } = useField(props.id);
     const fieldProps = useMemo(
       () => ({
         ...propGetters,
-        getMessageProps,
+        getInputProps: (options: any, describeOptions: any = {}) =>
+          getInputProps(options, { ...describeOptions, isDescribed: hasHint, hasMessage }),
+        getMessageProps: (options: any) => getMessageProps({ role: 'alert', ...options }),
         isLabelActive,
         setIsLabelActive,
         isLabelHovered,
         setIsLabelHovered,
-        multiThumbRangeRef,
-        getInputProps: (options: any, describeOptions: any) =>
-          getInputProps(options, { ...describeOptions, isDescribed: hasHint }),
-        setHint: (hintPresent: boolean) => setHasHint(hintPresent),
-        hasHint
+        hasHint,
+        setHasHint,
```

Comment:
`setHint` was renamed to `setHasHint` for consistency, the new `setHasMessage` also follows this naming pattern.